package.json

```diff
@@ -427,6 +419,22 @@
           "type": "string",
           "default": "latest",
           "markdownDescription": "Docker image tag to use when starting a local Schema Registry container. (By default, this will use the [`confluentinc/cp-schema-registry`](https://hub.docker.com/r/confluentinc/cp-schema-registry/tags) image.)"
+        },
+        "confluent.preview.enableDirectConnections": {
```

Comment:
cc @noeldevelops on the rename since you'll have to re-enable this setting for local development now that the old `confluent.experimental.enableDirectConnections` is no longer used